include/OpenColorIO/OpenColorIO.h

```diff
@@ -1149,64 +1142,49 @@ class OCIOEXPORT Config
-//   match and the next rule will be considered.
-//   \see FileRules::insertPathSearchRule.
-//   It has the key:
-//
-//   * name: Must be "ColorSpaceNamePathSearch".
-//
-// - Default Rule: The file_rules must always end with this rule. If no prior rules match,
-//   this rule specifies the color space applications will use.
-//   \see FileRules::setDefaultRuleColorSpace.
-//   It has the keys:
-//
-//   * name: must be "Default".
-//
-//   * colorspace : Color space name to be returned.
-//
-// Custom string keys and associated string values may be used to convey app or
-// workflow-specific information, e.g. whether the color space should be left as is
-// or converted into a working space.
-//
-// Getters and setters are using the rule position, they will throw if the position is not
-// valid. If the rule at the specified position does not implement the requested property
-// getter will return NULL and setter will throw.
-//
+/**
+ * \brief
+ * The File Rules are a set of filepath to color space mappings that are evaluated
+ * from first to last. The first rule to match is what determines which color space is
+ * returned. There are four types of rules available. Each rule type has a name key that may
+ * be used by applications to refer to that rule. Name values must be unique i.e. using a
+ * case insensitive comparison. The other keys depend on the rule type:
+ *
+ * * Basic Rule: This is the basic rule type that uses Unix glob style pattern matching and
+ *   is thus very easy to use. It contains the keys:
+ *   * name: Name of the rule
+ *   * colorspace: Color space name to be returned.
+ *   * pattern: Glob pattern to be used for the main part of the name/path.
+ *   * extension: Glob pattern to be used for the file extension. Note that if glob tokens
+ *     are not used, the extension will be used in a non-case-sensitive way by default.
+ * * Regex Rule: This is similar to the basic rule but allows additional capabilities for
+ *   power-users. It contains the keys:
+ *   * name: Name of the rule
+ *   * colorspace: Color space name to be returned.
+ *   * regex: Regular expression to be evaluated.
+ * * OCIO v1 style Rule: This rule allows the use of the OCIO v1 style, where the string
+ *   is searched for color space names from the config. This rule may occur 0 or 1 times
+ *   in the list. The position in the list prioritizes it with respect to the other rules.
+ *   StrictParsing is not used. If no color space is found in the path, the rule will not
+ *   match and the next rule will be considered.
+ *   see \ref insertPathSearchRule.
+ *   It has the key:
+ *   * name: Must be "ColorSpaceNamePathSearch".
+ * * Default Rule: The file_rules must always end with this rule. If no prior rules match,
+ *   this rule specifies the color space applications will use.
+ *   see \ref setDefaultRuleColorSpace.
+ *   It has the keys:
+ *   * name: must be "Default".
+ *   * colorspace : Color space name to be returned.
```

Comment:
I need to take another stab at this. 
 - I'd like to restore the newlines if possible, to improve readability
 - The sub-bullets aren't rendering as sub-bullets, they're rendering as regular bullets, which isn't too helpful.